doc/classes/VisualServer.xml

```diff
@@ -1951,6 +1951,11 @@
 			<argument index="2" name="scenario" type="RID">
 			</argument>
 			<description>
+				Returns an array of object IDs intersecting with the provided 3D ray. Only visual 3D nodes are considered, such as [MeshInstance] or [DirectionalLight].
+				Use [code]instance_from_id[/code] to obtain the actual nodes.
```

Comment:
Instead of [code]methodname[/code] You can use [method @GDscript.instance_from_id] to link directly to the function. It is described here https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html#improve-formatting-with-bbcode-style-tags